system-config/zvol_is_qubes_volume

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/bash
+
```

Comment:
`set -e` ?

Reply:
Every line that needed error handling has had it already.  We do not care if zfs list fails, because we evaluate the stdout, not the stderr.

Reply:
The point of `set -e` is about unexpected errors, not expected ones. For example if getting the dataset fails for some reason.

Reply:
I've added it.